src/passes/Print.cpp

```diff
@@ -1856,6 +1856,21 @@ struct PrintExpressionContents
     printMedium(o, "array.len ");
     printHeapTypeName(o, curr->ref->type.getHeapType());
   }
+  void visitRefAs(RefAs* curr) {
+    switch (curr->op) {
+      case RefAsFunc:
+        printMedium(o, "ref.as_func");
+        break;
+      case RefAsData:
+        printMedium(o, "ref.as_data");
+        break;
+      case RefAsI31:
+        printMedium(o, "ref.as_i31");
+        break;
+      default:
+        WASM_UNREACHABLE("unimplemented ref.is_*");
```

Comment:
```suggestion
        WASM_UNREACHABLE("invalid ref.is_*");
```
`unimplemented` sounds like we still have some instructions unimplemeneted?